mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1187,6 +1187,10 @@ and all pods running on the nodes. Specified as a map from the key, such as
 
 * `audit_mode` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Sets which mode of auditing should be used for the cluster's workloads. Accepted values are DISABLED, BASIC.
 
+<a name="nested_fleet"></a>The `fleet` block supports:
```

Comment:
you'll also need to add this field higher up in the doc (probably right under gateway_api_config and with a similar wording) and have it link to this section. You'll also need to add the output-only field to the "attributes reference" section below.